File: java-api-core/test/com/kinvey/java/model/DeleteResponseTest.java
package com.kinvey.java.model;

import junit.framework.TestCase;

/**
 * Created by edward on 7/31/15.
 */
public class DeleteResponseTest extends TestCase{

    KinveyDeleteResponse kdr;

    public void testConstructor(){
        kdr = new KinveyDeleteResponse();
        assertEquals(0, kdr.getCount());
        kdr.setCount(1);
        assertEquals(1, kdr.getCount());
    }
}

File: java-api-core/test/com/kinvey/java/model/DeleteResponseTest.kt
package com.kinvey.java.model

import junit.framework.TestCase

/**
 * Created by edward on 7/31/15.
 */
class DeleteResponseTest : TestCase() {

    var kdr: KinveyDeleteResponse? = null

    fun testConstructor() {
        kdr = KinveyDeleteResponse()
        assertEquals(0, kdr?.count)
        kdr?.count = 1
        assertEquals(1, kdr?.count)
    }
}